cdc/owner.go

```diff
@@ -615,7 +617,11 @@ func (o *Owner) watchCapture(ctx context.Context) error {
 		clientv3.WithPrevKV())
 
 	for resp := range ch {
-		if resp.Err() != nil {
+		err := resp.Err()
+		failpoint.Inject("restart-capture-watch", func() {
```

Comment:
seems we have no unit test for owner now

Reply:
Yes, the whole file is commented out temporarily. A unit test should be added after the file is recovered.